### DISCUSSION TITLE
How to install?

### DISCUSSION THREAD

**Discussion by KrystalYanBJ** [goldendict/goldendict#1479] [Q&A] [answered]
Hi! Can somebody please provide a clear instruction on how to install goldendict on Mac? The instructions under the code are obviously not written for a common user and I don't have any IT specialists among my friends to ask. Thanks in advance!

**Comment by ChihSee-Hsie** [comment, marked as answer]
Just visit [Here](https://github.com/goldendict/goldendict/wiki/Early-Access-Builds-for-Mac-OS-X), download the installation package according to the description and operate as what instructed. Actually what shall be really hard is the installation of dictionaries but not the software itself. I myself is also a newbie.

**Comment by xiaoyifang** [comment]


**Reply by vedgy** [reply]
This is a build of a GoldenDict fork, not a community-built version.
